lib/api/website.js

```diff
@@ -76,15 +79,32 @@ function _errorActions(err, errorDocument, routingRules,
     return callback(err, false, null, corsHeaders);
 }
 
+function capitalize(str) {
+    if (!str || typeof str !== 'string') return str;
+    return str.charAt(0).toUpperCase() + str.slice(1);
+}
+
+function callbackGetToHead(callback) {
+    return (err, userErrorPageFailure, dataGetInfo,
+        resMetaHeaders, redirectInfo, key) =>
+        callback(err, resMetaHeaders, redirectInfo, key);
+}
```

Comment:
Should I rather update the callback in Arsenal (https://github.com/scality/Arsenal/blob/development/7.10/lib/s3routes/routes/routeWebsite.ts#L57) to match HEAD and GET ?

Reply:
I don't know, do as you see fit :wink: I realized that the code was duplicated and mentioned it can be worth merging it, but I'll leave the implementation details to your discretion.

Reply:
Ok I'll leave it like that, I don't know if this arsenal part could be used somewhere else